trame/app/file_upload.py

```diff
@@ -22,36 +25,36 @@ def __init__(self, file_state_variable=None):
                 self._content = b"".join(self._content)
 
     @property
-    def is_empty(self):
+    def is_empty(self: ClientFile) -> bool:
         """Return true if the file is empty"""
         return self._content is None
 
     @property
-    def name(self):
+    def name(self: ClientFile) -> str | None:
         """File name"""
         return self._name
 
     @property
-    def size(self):
+    def size(self: ClientFile) -> int | None:
         """File size in Bytes"""
         return self._size
 
     @property
-    def modified_time(self):
+    def modified_time(self: ClientFile):
```

Comment:
Not sure what type is returned here.

Reply:
It is a number that correspond to the file time. You can test it with that example to be sure https://kitware.github.io/trame/examples/core/files.html#upload